src/tracking/TrackingEvent.ts

```diff
@@ -1,3 +1,20 @@
+export type TrackingFeatureName = '' |
+	'SoftClose' |
+	'Page' |
+	'AlreadyDonated' |
+	'CustomAmount' |
+	'BannerConductor' |
```

Comment:
I'd remove `BannerConductor` because it's the name of an internal component, that could be confusing for the campaigns team. Either leave it empty or use `Page`